### PR TITLE
Revert "Loggers are now created using utils.get_logger"

### PR DESCRIPTION
**Pull request by akx**
This mostly reverts commit 53e69feda77954b86b4b0bae5623bd85009826b3 (see discussion in #478).

(That commit also broke tests, since it's expecting to be run in a webui context.)

I'm of course open for discussion here, but I don't think forcing a log format and level here is the right option.

If the idea is that `dynamicprompts` and `sd_dynamic_prompts` log messages are visible, then we'd configure that with the `logging` hierarchical logger system separately somewhere.

### CODE REVIEW

**Comment by adieyal** [comment]
I'm not super wedded to this, it's only that it isn't always clear where a particular message comes from.

**Comment by akx** [comment]
> I'm not super wedded to this, it's only that it isn't always clear where a particular message comes from.

That's why each module's logger should be instantiated with `logging.getLogger(__name__)` (or by spelling `__name__` out by hand, which may be required for webui extensions).

File: sd_dynamic_prompts/dynamic_prompting.py [hunk 1]
```diff
@@ -28,12 +29,16 @@
...
 from sd_dynamic_prompts.pnginfo_saver import PngInfoSaver
 from sd_dynamic_prompts.prompt_writer import PromptWriter
-from sd_dynamic_prompts.utils import get_logger
 
 VERSION = __version__
 
-logger = get_logger(__name__)
-logger.info(f"Starting Dynamic Prompts v{VERSION}")
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
```
Comment (adieyal) [inline sd_dynamic_prompts/dynamic_prompting.py:36]: Given your thoughts on the module setting its own level - shouldn't this line be removed?

File: sd_dynamic_prompts/dynamic_prompting.py [hunk 2]
```diff
@@ -28,12 +29,16 @@
...
+
+is_debug = getattr(opts, "is_debug", False)
+
+if is_debug:
+    logger.setLevel(logging.DEBUG)
```
Comment (adieyal) [inline sd_dynamic_prompts/dynamic_prompting.py:41]: ditto

**Pull request merged**